fix-install.sh

```diff
@@ -3,8 +3,9 @@
 set -e
 set -u
 
-cd $cur__lib/melange
-tar xvf libocaml.tar.gz
+mkdir -p $cur__lib/ocaml
```

Comment:
Does it makes sense to call `ocaml` it's not only the OCaml stdlib here

Reply:
This also needs to be updated https://github.com/melange-re/melange/blob/fork/esy.json#L32